apps/client/src/pages/Login/LoginComponents.tsx

```diff
@@ -0,0 +1,124 @@
+import {
+  Heading,
+  useAuthenticator,
+  useTheme,
+  View,
+} from '@aws-amplify/ui-react';
+import { Button } from '@cloudscape-design/components';
+
+export const loginComponents = {
+  SignIn: {
+    Header() {
+      const { tokens } = useTheme();
+
+      return (
+        <Heading
+          padding={`${tokens.space.xl.value} 0 0 ${tokens.space.xl.value}`}
+          level={3}
+        >
+          Sign in to your account
+        </Heading>
+      );
+    },
+    Footer() {
+      const { tokens } = useTheme();
+      const { toResetPassword } = useAuthenticator();
+
+      return (
+        <View padding={`0 0 ${tokens.space.xl.value} 0`} textAlign="center">
+          <Button onClick={toResetPassword}>Reset Password</Button>
+        </View>
+      );
+    },
+  },
+
+  SignUp: {
+    Header() {
+      const { tokens } = useTheme();
+
+      return (
+        <Heading
+          padding={`${tokens.space.xl.value} 0 0 ${tokens.space.xl.value}`}
+          level={3}
+        >
+          Create a new account
+        </Heading>
+      );
+    },
+    Footer() {
+      const { tokens } = useTheme();
+      const { toSignIn } = useAuthenticator();
+
+      return (
+        <View padding={`0 0 ${tokens.space.xl.value} 0`} textAlign="center">
+          <Button onClick={toSignIn}>Back to Sign In</Button>
+        </View>
+      );
+    },
+  },
+  ConfirmSignUp: {
+    Header() {
+      const { tokens } = useTheme();
+      return (
+        <Heading
+          padding={`${tokens.space.xl.value} 0 0 ${tokens.space.xl.value}`}
+          level={3}
+        >
+          Enter Information:
+        </Heading>
+      );
+    },
+  },
+  SetupTOTP: {
+    Header() {
+      const { tokens } = useTheme();
+      return (
+        <Heading
+          padding={`${tokens.space.xl.value} 0 0 ${tokens.space.xl.value}`}
+          level={3}
+        >
+          Enter Information:
+        </Heading>
+      );
+    },
+  },
+  ConfirmSignIn: {
+    Header() {
+      const { tokens } = useTheme();
+      return (
+        <Heading
+          padding={`${tokens.space.xl.value} 0 0 ${tokens.space.xl.value}`}
+          level={3}
+        >
+          Enter Information:
+        </Heading>
+      );
+    },
+  },
+  ResetPassword: {
+    Header() {
+      const { tokens } = useTheme();
+      return (
+        <Heading
+          padding={`${tokens.space.xl.value} 0 0 ${tokens.space.xl.value}`}
+          level={3}
+        >
+          Enter Information:
+        </Heading>
+      );
+    },
+  },
+  ConfirmResetPassword: {
+    Header() {
+      const { tokens } = useTheme();
+      return (
+        <Heading
+          padding={`${tokens.space.xl.value} 0 0 ${tokens.space.xl.value}`}
```

Comment:
blocking: use components directly when possible, rather than design tokens. should be able to achieve this with box